### PR TITLE
Initialize minimal working training routine

### PR DESCRIPTION
**Pull request by mmbajo**
## What
* Minimal working training routine

## TODO
- [ ] Load reazonspeech small dataset
- [ ] Make loading small dataset fraction for hyperparam search and debugging
- [ ] Decide the split for dev and training set
- [ ] Script for loading of test set JSUT

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        model_args, data_args, training_args = parser.parse_args_into_dataclasses()
+        
+    # setting seed for reproducibility
+    set_seed(training_args.seed)
+    
+    # load datasets
+    raw_datasets = DatasetDict()
+    if training_args.do_train:
+        raw_datasets["train"] = load_datasets_from_config(
+            "conf/dataset_config.yaml",
```
Comment (mmbajo) [inline main.py:40]: 📝 Bad practice but please let me hardcode the dataset config for now. Will add this to the `args.py` in a later PR. 🙇

File: src/metrics.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import evaluate
+from transformers.models.whisper.english_normalizer import BasicTextNormalizer
+import MeCab
+
+
+class TextNormalizer:
+    def __init__(self):
+        self.wakati = MeCab.Tagger("-Owakati")
```
Comment (mmbajo) [inline src/metrics.py:8]: 📝 Segmentation of Japanese sentences using MeCab

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "tokyo-whisperers"
+version = "0.1.0"
+description = ""
+authors = ["mmbajo <markarjaybajo@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+reazonspeech-nemo-asr = {git = "https://github.com/reazon-research/ReazonSpeech", subdirectory = "pkg/nemo-asr"}
```
Comment (mmbajo) [inline pyproject.toml:10]: 📝 ReazonSpeech models is defined in our environement. Once installed via `poetry install` you can call the nemo model

https://research.reazon.jp/projects/ReazonSpeech/api/reazonspeech.nemo.asr.html#
